keps/sig-node/3545-improved-multi-numa-alignment/README.md

```diff
@@ -302,6 +303,12 @@ When an option graduates, its visibility should be moved to be controlled by the
 The introduction of these feature gates gives us the ability to move the option to beta and later stable without implying that all available options are stable.
 This approach is similliar to graduation criteria for `CPUManagerPolicyOptions` introduced [here](https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/2625-cpumanager-policies-thread-placement#graduation-criteria-of-options).
 
+In 1.28 this feature is being promoted to Beta. We propose following changes to TopologyManager policy options default visibility:
+
+- `TopologyManagerPolicyOptions` feature flag for enabling/disabling the entire feature will be enabled by default.
+- `TopologyManagerPolicyBetaOptions` feature flag for enabling/disabling beta options will be enabled by default.
```

Comment:
Why is TopologyManagerPolicyOptions enabled by default for beta stage?

Reply:
I was following the path of `CPUManagerPolicyOptions` for its beta graduation  [here](https://github.com/kubernetes/kubernetes/commit/077c0aa1be079d59e97cac50f44b64244590135b#diff-71e3b98f9a6bbf5b8421e26a7ba0c079f397cd8d49abacdad943c66a4f44f03dR883). I was also under impression that beta stage feature gates are enabled by default but if that's not the case I can change it so it'll be disabled.